cmds/bmp/bms_command.go

```diff
@@ -1,9 +1,19 @@
 package bmp
 
 import (
+	"errors"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+	"io/ioutil"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
 	clients "github.com/cloudfoundry-community/bosh-softlayer-tools/clients"
 	cmds "github.com/cloudfoundry-community/bosh-softlayer-tools/cmds"
 	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	tablewriter "github.com/olekukonko/tablewriter"
```

Comment:
I'll merge with what you have now but if the @cppforlife is better then let's look into switching to it as a chore. Especially since I know DK's is being used or will be used in BOSH code and might have coloring which would be nice.